pytest_django/plugin.py

```diff
@@ -481,6 +481,28 @@ def django_db_blocker():
     return _blocking_manager
 
 
+@pytest.fixture(scope="session")
+def django_db_blocker(django_db_setup, _django_db_blocker):
+    """Wrapper around _django_db_blocker to serve as convenience reference.
+
+    The ``_django_db_blocker`` fixture must be available for the ``django_db_setup``
+    fixture, so ``django_db_setup`` must request the ``_django_db_blocker`` fixture. But
+    in order for ``_django_db_blocker`` to be used, ``django_db_setup`` must also have
+    been executed, suggesting that ``_django_db_blocker`` should request
+    ``django_db_setup``, especially since it is possible for ``_django_db_blocker`` to
+    be needed when ``django_db_setup`` wouldn't normally have been run (e.g. if a test
+    isn't marked with ``pytest.mark.django_db``).
```

Comment:
> especially since it is possible for ``_django_db_blocker`` to
￼    be needed when ``django_db_setup`` wouldn't normally have been run (e.g. if a test
￼    isn't marked with ``pytest.mark.django_db``).

This now implicitly "marks" the test then, doesn't it?

Reply:
I believe it's _almost_ the same. The only difference being that the DB is still blocked if you don't use the mark.